lib/phoenix_live_controller.ex

```diff
@@ -637,10 +630,18 @@ defmodule Phoenix.LiveController do
       def message_handler(socket, name, message),
         do: apply(__MODULE__, name, [socket, message])
 
+      def view_module,
+        do:
+          __MODULE__
+          |> to_string()
+          |> String.replace(~r/(Live|LiveController|Controller)?$/, "View", global: false)
```

Comment:
With this change we cannot be sure anymore that the View suffix will be added at all e.g. for live view named `MyAppWeb.SomethingCool`. We should somehow check if after this call there's "View" at the end and add it if there's not.